mbtools/validate_mbz_html.py

```diff
@@ -51,10 +59,11 @@ def find_style_violations(html_elements):
     for elem in html_elements:
         attributes = elem.get_attribute_values("style")
         for attr in attributes:
-            if attr not in VALID_STYLES:
+            if attr not in VALID_STYLES and attr != "":
```

Comment:
Clarification question: This will now basically flag all non-empty instances of the `style` attribute, correct? If so, I assume the idea is: 1) We would continue to be able to see how loud this is / what styles are being incorporated into content; 2) It would be easy to filter those rows out of the CSV for WW.

Reply:
Yes, that'd to what I had in mind. It's very easy to filter out all of the style violations or, with a small code tweak, filter them out by specific types.We can use this as a team to determine which ones to wipe away and which to keep.